docker-compose.example.yml

```diff
@@ -5,46 +5,24 @@ services:
     image: unocha/mongo:4.0.5
     volumes:
       - ./db:/srv/db
-    # environment:
-    #   DNSDOCK_NAME: db
-    #   DNSDOCK_IMAGE: hid
   dev:
     image: unocha/nodejs:8.11.3
     volumes:
       - .:/srv/www
     ports:
       - 0.0.0.0:3000:3000
     environment:
-      # DNSDOCK_NAME: api
-      # DNSDOCK_IMAGE: hid
-      # PORT: 80
       NODE_ENV: local
-      ROOT_URL: http://api.hid.vm
-      SMTP_HOST: app.mailhog.vm
+      ROOT_URL: http://localhost:3000
+      SMTP_HOST: mailhog
       SMTP_PORT: 1025
-      SMTP_USER: humanitarianid
-      SMTP_PASS: smtp_password
-      APP_URL: http://app.hid.vm
+      SMTP_USER: 
```

Comment:
Does this intentionally remove a user/pass?

Reply:
it makes possible to actually use the SMTP_USER and SMTP_PASS in the `.env` file to define it for the desired environment. Providing a hardcoded default was useless anyway.